gatsby/src/includes/set-user/javascript.mdx

```diff
@@ -0,0 +1,5 @@
+```javascript
```

Comment:
if this renders fine, no change. Or should it be "```js"?

Reply:
i believe behind the scenes its just an alias, so you can use either